### PR TITLE
Issue/9511 fix followers crash

### LINKED ISSUE #9511: Stats: Crash when opening the View all Followers
Stack trace:
```
Fatal Exception: kotlin.UninitializedPropertyAccessException: lateinit property siteModel has not been initialized
       at org.wordpress.android.ui.stats.refresh.utils.StatsSiteProvider.getSiteModel(StatsSiteProvider.java:17)
       at org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases.FollowersUseCase.loadCachedData(FollowersUseCase.java:65)
       at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase.fetch(BaseStatsUseCase.java:78)
       at org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase$fetch$1.invokeSuspend(BaseStatsUseCase.java)
       at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(BaseContinuationImpl.java:32)
       at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.java:233)
       at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.java:594)
       at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.java:60)
       at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.java:742)
```

### PR DESCRIPTION
**Pull request by 0nko**
Fixes #9511.

To test:
1. Go to Notifications tab.
2. Tap Follows tab.
3. Scroll to bottom of list.
4. Tap See all of your followers button link.
5. Notice a list of all followers is loaded

### CODE REVIEW

**Comment by malinajirka** [comment]
I'm sorry, but I'm a bit behind the schedule, so it'd be great if someone else could review it. Thanks! (or I can review it next week.)

**Comment by jtreanor** [comment]
👋 it would be great if we could get this reviewed/merged today so I can do final 12.1 beta this week. @AmandaRiu maybe you could take a look at this?

**Comment by malinajirka** [comment]
Ohh, sorry - I missed it's a fix for 12.1. I'm starting the review now ;).

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllViewModel.kt [hunk 1]
```diff
@@ -83,7 +88,11 @@ class StatsViewAllViewModel(
 
     private suspend fun loadData(refresh: Boolean, forced: Boolean) {
         withContext(bgDispatcher) {
-            useCase.fetch(refresh, forced)
+            if (statsSiteProvider.hasLoadedSite()) {
+                useCase.fetch(refresh, forced)
+            } else {
+                mutableSnackbarMessage.value = R.string.stats_site_not_loaded_yet
```
[the remark this answers is not in the document]
Comment (0nko) [reply WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllViewModel.kt]: Good catch!

File: WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllViewModel.kt [hunk 2]
```diff
@@ -60,6 +60,11 @@ class StatsViewAllViewModel(
 
     val toolbarHasShadow = dateSelectorData.map { !it.isVisible }
 
+    fun start(site: SiteModel) {
```
[the remark this answers is not in the document]
Comment (0nko) [reply WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewAllViewModel.kt]: I agree.

**Comment by jtreanor** [comment]
Thanks everyone for wrangling this!

**Pull request merged**